<https://rt.cpan.org/Public/Bug/Display.html?id=76442>

It has been predicted that D:C:XS would be affected but I did not read
it. The following patch works:

--- src/C_XS/ToolBox.h       2009-10-16
23:47:55.000000000 +0200
+++ src/C_XS/ToolBox.h        2012-04-09
14:27:56.116897462 +0200
@@ -93,7 +93,10 @@
     #elif PERL_DARWIN
         #define boolean bool
     #else
-        typedef enum { false = FALSE, true = TRUE } boolean;
+        typedef int boolean;
+        #ifndef I_STDBOOL
+         enum { false, true };
+        #endif 
     #endif
 #endif
 


Tested with v5.15.2-436-gbd31be4 and v5.15.9-123-gbd9cbc4

